python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -564,7 +605,8 @@ class RandomForestClassifier(Base):
     def predict(self, X, predict_model="GPU",
                 output_class=True, threshold=0.5,
                 algo='BATCH_TREE_REORG',
-                num_classes=2, convert_dtype=True):
+                num_classes=2, convert_dtype=True,
+                concat_mod_bytes=[]):
```

Comment:
Is there not a better way to not change the api and make this confusing for users? Even using **kwargs might be better than having this parameter that is detrimental to the normal predict api